site/content/internal/onboarding/new-staff-guide.md

```diff
@@ -24,20 +24,20 @@ weight: 10
 - List to be sent by manager in channel
 
 ### Channels and Teams
-- You should be added to the private team `Private Core` and join the following channels
+- You should be added to the private team `Staff` and join the following channels
     - Alerts
     - Announcements
-    - Community/Customers
+    - Customer Success
+    - Customer Support
+    - Confidential Bugs
     - New CLA Entries
-    - Platform Meeting
     - Private Off-Topic (optional)
     - Social: * (optional)
-    - Standup
+    - Stand-up
     - Welcome
 - Private channels to join in `Contributors`
     - Security
-    - Developer: Private
-    - Confidential Bugs
+    - Developers: Private     
```

Comment:
You should also join `MVP Discussion`.

Reply:
I would propose not adding MVP Discussion. It's a private channel so people can't join it. I add new R&D folks when they are announced.